osu.Game/Screens/Play/FailAnimation.cs

```diff
@@ -136,6 +144,8 @@ public void Start()
             Content.ScaleTo(0.85f, duration, Easing.OutQuart);
             Content.RotateTo(1, duration, Easing.OutQuart);
             Content.FadeColour(Color4.Gray, duration);
+
+            Background?.FadeColour(OsuColour.Gray(0.3f), 60);
```

Comment:
Slightly uneasy about the onus of undoing this effect being on the next screen which is going to be resumed, but as far as I can tell this doesn't break anything (and I've tested several combinations, typical and atypical), so...

Reply:
I agree with this sentiment. It looks like an error and I had to run the branch to make sure it works.